### PR TITLE
Made goal input error orange

### LINKED ISSUE #878: [Redesign assignmnent] Make goal box error orange, not red
Need to set this in the theme object.

### PR DESCRIPTION
**Pull request by justin0022**
Resolves #878

### CODE REVIEW

**Comment by ssciolla** [comment]
Sorry, @justin0022, I've been distracted with other projects. I'll take a look at this next week. We may want to consider whether we should align the orange we use here with the yellow/orange in the `WarningBanner`.

File: assets/src/components/StyledTextField.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
 import TextField from '@material-ui/core/TextField'
 import { withStyles } from '@material-ui/core/styles'
 
+const orange = '#ffae42'
+
 const StyledTextField = withStyles({
   root: {
     '& .MuiFormLabel-root.Mui-error': {
-      color: '#ffae42'
+      color: orange
```
Comment (ssciolla) [inline assets/src/components/StyledTextField.js:9]: @justin0022, can you look into setting and utilizing the `warning` color in the theme palette? I think we can override the default `warning`, maybe with what we have in `WarningBanner` (`"#FFF3CD"`?). Maybe that's too light, but seems like we should somehow standardize uses of orange in the application.

Palette: https://material-ui.com/customization/palette/#palette-colors

Comment (ssciolla) [reply assets/src/components/StyledTextField.js:9]: Maybe whatever you use for the `dark` part could be what you use in this component.

Comment (justin0022) [reply assets/src/components/StyledTextField.js:9]: Sure, will do!

**Comment by ssciolla** [comment]
Hi @justin0022, just FYI, I'm going to finish this up.

**Comment by justin0022** [comment]
> Hi @justin0022, just FYI, I'm going to finish this up.

Thanks Sam, sorry I haven't had a chance to work on this.

**Comment by ssciolla** [comment]
> Thanks Sam, sorry I haven't had a chance to work on this.

No worries, @justin0022, I'm closing this PR; please see #1154.

**Pull request closed without merging**